src/ml/neural_net/mps_compute_context.mm

```diff
@@ -88,12 +88,45 @@
 std::unique_ptr<model_backend> mps_compute_context::create_object_detector(
     int n, int c_in, int h_in, int w_in, int c_out, int h_out, int w_out,
     const float_array_map& config, const float_array_map& weights) {
-
+  float_array_map updated_config;
+  constexpr float MPS_LOSS_MULTIPLIER = 8;
+  if (config.count("learning_rate") > 0) {
+    updated_config["learning_rate"] =
+      shared_float_array::wrap(*config.at("learning_rate").data() / MPS_LOSS_MULTIPLIER);
+  }
```

Comment:
Could we define a quicker helper function or lambda here?

Also, prefer this pattern:
```
auto config_iter = config.find("learning_rate");
if (config_iter != config.end()) {
  float old_value = config_iter->second.data()[0];
  float new_value = old_value / MPS_LOSS_MULTIPLIER;
  config_iter->second = shared_float_array::wrap(new_value);
}